External/FEXCore/Source/Interface/Core/JIT/x86_64/JIT.cpp

```diff
@@ -875,6 +888,14 @@ void JITCore::CreateCustomDispatch(FEXCore::Core::InternalThreadState *Thread) {
     cmp(rax, 0);
     je(NoBlock);
 
+    // Update L1
+    mov(r13, Thread->BlockCache->GetL1Pointer());
+    mov(rcx, rdx);
+    and_(rcx, 1 * 1024 * 1024 - 1);
```

Comment:
Don't use hard coded L1C size.
Pull a constant from the BlockCache class instead

Reply:
Done